cloudferrylib/os/network/neutron.py

```diff
@@ -279,7 +279,6 @@ def convert_subnets(snet, cloud):
         }
 
         res_hash = network_res.get_resource_hash(result,
-                                                 'name',
```

Comment:
Yes, but having "name" here gives more chances our subnet comparison is correct, because tenant can have more than one subnet with the same CIDR. For example having name will handle following case correctly, while without name it will be invalid:
```
# as tenant t1
neutron net-create net1
neutron subnet-create --name subnet1 net1 10.0.0.0/24
neutron net-create net2
neutron subnet-create --name subnet1 net2 10.0.0.0/24
```